### ISSUE TITLE
Update NERC Documentation (MB/GB/TB -> MiB/GiB/TiB)

### ISSUE THREAD

**Issue by joachimweyl** [nerc-project/operations#587]
## Motivation
currently we are displaying MB/GB/TB when technically we are using MiB/GiB/TiB.

## Completion Criteria
Update customer facing documents to display what we are actually using.

## Description
- [x] Update [Invoicing Requirements](https://docs.google.com/document/d/1SHCMidWQmazACGqzYOlRHQqnsg4KjmeW8ShGiEptR2Q/edit?usp=sharing)
- [x] Update [Calculator](https://docs.google.com/spreadsheets/d/187bQgz3eB3W9OL7x9rrgaJ2pi7u85xMJvUJbnKzzBQM/edit?usp=sharing)
- [x] Update ColdFront Quota Attributes for OpenStack and OpenShift along with SU label
- [x] Update NERC Web documents

## Completion dates
Desired - 2024-05-29
Required - TBD

**Comment by knikolla** [comment]
For storage, we are charging based on quota.

For object storage, we are setting the quota[[0]](https://github.com/nerc-project/coldfront-plugin-cloud/blob/ae311f876143a78ad822f6398ad37b134283279f/src/coldfront_plugin_cloud/openstack.py#L22) per GB.

For volume storage, we are setting the quota on GiB for both OpenStack[[1]](https://docs.openstack.org/api-ref/block-storage/v3/#show-quota-classes-for-a-project)(as that is the only unit accepted) and OpenShift[[2]](https://github.com/nerc-project/coldfront-plugin-cloud/blob/ae311f876143a78ad822f6398ad37b134283279f/src/coldfront_plugin_cloud/openshift.py#L15-L16).

When it comes to charging, the advertised price in the docs is written per TB. Until now, I have been dividing that price of $0.009 by 1000 to come up for the per GB value.

That means that for object storage we are charging per TB of storage with the advertised price of $0.009/TB, whereas for volume storage we are charging $0.009/1000GiB.

We should update the documentation and investigate a path to consistency.

0. https://github.com/nerc-project/coldfront-plugin-cloud/blob/ae311f876143a78ad822f6398ad37b134283279f/src/coldfront_plugin_cloud/openstack.py#L22
1. https://docs.openstack.org/api-ref/block-storage/v3/#show-quota-classes-for-a-project
2. https://github.com/nerc-project/coldfront-plugin-cloud/blob/ae311f876143a78ad822f6398ad37b134283279f/src/coldfront_plugin_cloud/openshift.py#L15-L16

**Comment by knikolla** [comment]
@joachimweyl @Milstein  Let me know if I captured correctly the outcome of May 22, 2024 meeting

- Pricing advertised in rates page is per TiB rather than per TB, so the price per GiB is adjusted to $0.009/1024 from $0.009/1000 that it currently is in the billing scripts. 
- We should move to GiB for object storage too. Need to create an issue to capture this code change and migration.
- This will be effective for the June billing cycle.

**Comment by Milstein** [comment]
@knikolla @joachimweyl: Can we confirm this statement is correct in our nerc docs:

> Storage is charged separately at a rate of $0.009 TiB/hr or $9.00E-12 KiB/hr at a granularity of KiB/hr. For ease of reporting, storage SUs are shown in GiB but calculated in KiB.

**Comment by knikolla** [comment]
> @knikolla @joachimweyl: Can we confirm this statement is correct in our nerc docs:
> 
> > Storage is charged separately at a rate of $0.009 TiB/hr or $9.00E-12 KiB/hr at a granularity of KiB/hr. For ease of reporting, storage SUs are shown in GiB but calculated in KiB.

- The granularity is at the level of GiB. Storage SUs are calculated in GiB. I'm not aware of anywhere we're converting those values to KiB.  
- We are currently billing as I described in the comment above, and starting with June we will be billing by what you describe here, yes.

**Comment by joachimweyl** [comment]
documents updated, waiting on some screenshots. the PR will automatically close this issue.

**Comment by joachimweyl** [comment]
@Milstein can I be of assistance in gathering any of the parts you need?

**Comment by Milstein** [comment]
Upcoming NERC's RegApp & ColdFront service upgrade and maintenance
[June 18, 2024 9:00 AM - 11:00 AM ET]) will include most of these changes except the screenshots update for the nerc docs.

**Comment by joachimweyl** [comment]
@Milstein let's close this out on Tuesday once the documents are live and create a new issue to track adding the screenshots.
